site/gatsby-site/pages/downloadIndex.js

```diff
@@ -0,0 +1,166 @@
+import { useEffect } from 'react';
+import { useStaticQuery, graphql } from 'gatsby';
+
+const ALL_INCIDENTS_DATA = graphql`
+  query AllIncidentsDataQuery {
+    allMongodbAiidprodIncidents {
+      nodes {
+        mongodb_id
+        description
+        authors
+        image_url
+        language
+        source_domain
+        text
+        title
+        url
+        date_downloaded
+        date_modified
+        date_published
+        incident_date
+        epoch_date_downloaded
+        epoch_date_modified
+        epoch_date_published
+        epoch_incident_date
+        epoch_date_submitted
+        submitters
+        date_submitted
+        report_number
+        incident_id
+        ref_number
+      }
+    }
+    allMongodbAiidprodClassifications(filter: { namespace: { eq: "CSET" } }) {
+      nodes {
+        incident_id
+        id
+        namespace
+        classifications {
+          Annotation_Status
```

Comment:
We will likely re-add some of these later, but I think only the following should be here at the moment:

```
Harm_distribution_basis
Intent
Lives_lost
Location
Named_entities
Near_miss
Severity
Technology_purveyor
AI_Applications
AI_techniques
Finacial_Cost
Harm_Distribution_Basis
Harm_Type
Infrastructure_Sectors
Level_of_Autonomy
Lives_Lost
Named_Entities
Nature_of_End_User
Near_Miss
Physical_System
Problem_Nature
Public_Sector_Deployment
Relevant_AI_functions
Sector_of_Deployment
System_Developer
Technology_Purveyor
```

note that Finacial_Cost is misspelled.